test/curry.js

```diff
@@ -66,3 +66,20 @@ metatests.testSync('multiple curry of sum(x, y, z)', (test) => {
   test.strictSame(addTwoThree(10), 15);
   test.strictSame(addTwoThree(20), 25);
 });
+
+metatests.testSync('curry of identity', (test) => {
+  const id = x => x;
+  const idCurry = common.curry(id);
+  test.strictSame(idCurry(10), 10);
+  test.strictSame(idCurry(20), 20);
+
+  test.strictSame(common.curry(id, 10), 10);
+});
+
+metatests.testSync('curry of unit', (test) => {
+  const unit = () => 42;
+  const unitCurry = common.curry(unit);
+  test.strictSame(unitCurry(), 42);
+  test.strictSame(unitCurry(10), 42);
```

Comment:
I don't think this test and the next one make much sense theoretically, it's just how JavaScript inherently works and not a part of the spec of the tested function.

That said, I'm not strongly opposed to it, so if there already are some tests that call a curried function with more arguments than it's supposed to have, let's keep these too for consistency. If there are none yet, I'm -0 on adding them.

Reply:
Also, if we were to add these, I'd probably favor an explicit unit value, like `undefined`, instead of `10`. This, at least, can have some far-fetched theoretical justification of why it makes sense.